index.js

```diff
@@ -9,29 +9,30 @@ const VersionChecker = require('ember-cli-version-checker');
 
 const minEmberVersion = '3.0.0-beta.3';
 const flagToEnvironment = {
-  'ember-k': '_ENABLE_EMBER_K_SUPPORT',
-  'safe-string': '_ENABLE_SAFE_STRING_SUPPORT',
-  'enumerable-contains': '_ENABLE_ENUMERABLE_CONTAINS_SUPPORT',
-  'underscore-actions': '_ENABLE_UNDERSCORE_ACTIONS_SUPPORT',
-  'reversed-observer': '_ENABLE_REVERSED_OBSERVER_SUPPORT',
-  'initializer-arity': '_ENABLE_INITIALIZER_ARGUMENTS_SUPPORT',
-  'router-resource': '_ENABLE_ROUTER_RESOURCE',
-  'current-when': '_ENABLE_CURRENT_WHEN_SUPPORT',
-  'controller-wrapped': '_ENABLE_CONTROLLER_WRAPPED_SUPPORT',
-  'application-registry': '_ENABLE_DEPRECATED_REGISTRY_SUPPORT',
-  'immediate-observer': '_ENABLE_IMMEDIATE_OBSERVER_SUPPORT',
-  'string-fmt': '_ENABLE_STRING_FMT_SUPPORT',
-  'ember-freezable': '_ENABLE_FREEZABLE_SUPPORT',
-  'component-defaultlayout': '_ENABLE_COMPONENT_DEFAULTLAYOUT_SUPPORT',
-  'ember-binding': '_ENABLE_BINDING_SUPPORT',
-  'input-transform': '_ENABLE_INPUT_TRANSFORM_SUPPORT',
-  'deprecation-options': '_ENABLE_DEPRECATION_OPTIONS_SUPPORT',
-  'orphaned-outlets': '_ENABLE_ORPHANED_OUTLETS_SUPPORT',
-  'warn-options': '_ENABLE_WARN_OPTIONS_SUPPORT',
-  'resolver-function': '_ENABLE_RESOLVER_FUNCTION_SUPPORT',
-  'init-attrs': '_ENABLE_DID_INIT_ATTRS_SUPPORT',
-  'render-support': '_ENABLE_RENDER_SUPPORT',
-  'property-required': '_ENABLE_PROPERTY_REQUIRED_SUPPORT'
+  "ember-k": "_ENABLE_EMBER_K_SUPPORT",
```

Comment:
Seems odd to change all of these to double quotes, any particular reason?

Reply:
It was done automatically by the prettier VS extension. I can reverte it, but if we intend to use pretties across the board I'll leave it.

Reply:
Seems weird that prettier would only convert a small section of this file (there are a bunch of other single quoted things)...

Reply:
I reverted this bit.